_template/cmd/{{.Base.appName}}/main.go

```diff
@@ -18,8 +18,15 @@ func main() {
 	}
 }
 
+func getLogger() (*zap.Logger, error) {
+	if os.Getenv("LOG_LEVEL") == "debug" {
+		return zap.NewDevelopment()
+	}
+	return zap.NewProduction()
+}
```

Comment:
I'd prefer to really set the log level in the logger instead of differentiating between development and production logger with the `LOG_LEVEL` env var.

WDYT about sth like the following:
```suggestion
func getLogger() (*zap.Logger, error) {
	logLevel := zapcore.InfoLevel
	if levelStr := os.Getenv("LOG_LEVEL"); levelStr != "" {
		var err error
		logLevel, err = zapcore.ParseLevel(levelStr)
		if err != nil {
			return nil, err
		}
	}

	logConf := zap.NewProductionConfig()
	logConf.Level = zap.NewAtomicLevelAt(logLevel)

	logger, err := logConf.Build()
	if err != nil {
		return nil, err
	}

	return logger, nil
}
```